src/app/components/applications/SSV/OperatorSuccessPage/OperatorSuccessPage.styles.ts

```diff
@@ -18,6 +18,8 @@ export const useStyles = makeStyles((theme: Theme) => ({
     marginBottom: 24,
   },
   BoxWrapper: {
+    display: 'flex',
+    alignItems: 'center',
```

Comment:
please adjust order from shortest to longest

Reply:
done